src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,34 @@
+package edu.kis.vh.nursery;
+
+
+import edu.kis.vh.nursery.list.IntArrayStack;
+import edu.kis.vh.nursery.list.IntLinkedList;
+
+public class DefaultCountingOutRhymer {
+    IntLinkedList arr;
```

Comment:
Should be private.
The goal of the lab was to make using IntLinkedList and IntArrayStack interchangeable. You should use the interface where it is possible.
The name of variables shouldn't indicate the data structure type (arr -> stack)

There should be a constructor that takes ListInterface.